specs/IR-0003/constitution.md

```
@@ -0,0 +1,343 @@
+# Cloudbird-Software 宪法 v2（最终 Strawman，待 owner 审查）
+>
+> 状态：**signed v2.2**（randypanding，2026-08-21）。本文件为宪法正本，IR 见 #161。
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)